modules/github-events/internal/trampoline/server.go

```diff
@@ -78,16 +78,22 @@ func (s *Server) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		}
 	}
 
+	// Unmarshal payload to extract necessary information
+	var parsedPayload map[string]interface{}
```

Comment:
Could we make this the actual struct with fields we know we need?

`map[string]interface{}` is a bit harder to use (casting stuff to `string` like one line 88`) and it means we store all the deserialized data, not just the parts we need.

I normally wouldn't care so much about optimizing stuff like this, but this service is implicated by _every PR event_, and PR events can be pretty huge. If we're not careful we may wake up in a month to find this service requires a ton of memory and replicas.

Reply:
sounds good to me!